migrations/versions/5551af72812f_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 5551af72812f
+Revises: 6e5c574cbfb8
+Create Date: 2019-04-04 23:16:27.618573
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '5551af72812f'
+down_revision = '6e5c574cbfb8'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('access_codes', sa.Column('ticket_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'access_codes', 'tickets', ['ticket_id'], ['id'], ondelete='CASCADE')
```

Comment:
Why were so many foreign keys being generated?

Reply:
@iamareebjamal Was trying to debug but I believe it might be some error on my side of the system. Don't know the exact reason. Applied the migration again after pulling changes and it worked fine.